src/lib/remark-plugins/rewrite-tutorial-links/index.ts

```diff
@@ -27,8 +27,23 @@ import {
   getTutorialMap,
   handleCollectionLink,
   handleTutorialLink,
+  handleDocsLink,
 } from './utils'
 
+let TUTORIAL_MAP
+
+// @TODO - should this live in the __config data?
```

Comment:
Open to thoughts here!

Reply:
I think that would probably be a good idea, or something like `src/constants`. We've got a few different flavors of this map sprinkled throughout dev-portal and our other repos.

Reply:
I'll make a ticket and defer doing that in this PR 👌

Reply:
https://app.asana.com/0/1201987349274776/1202184785235838